docs/acceptance-mobile.md

```diff
@@ -0,0 +1,83 @@
+# Send V2 UX Mobile Acceptance and Spec Annotations
+
+`Date Created: 8/20/2018`
+
+## Acceptance Criteria
+
+Adapted from [this spreadsheet](https://airtable.com/shrkcBPOLkvNFOrpp)
+
+- [ ] It should look and feel of an Android App
+- [ ] It should look and feel like the Firefox Send Web Client
+
+### Main Screen
+- [ ] It should clearly Indicate the name of the product
+- [ ] If user has no existing Sends, it should make clear the primary benefits of the service (private, e2e encrypted, self-destructing file sharing)
+- [ ] It should allow users to access the file picker to create Send links
+- [ ] If the user has existing Sends, it should display a card-based list view of each [see Cards section below]
+
+### Non-Authenticated Users
+- [ ] It should make clear the benefits of a Firefox Account
+- [ ] It should allow users to log into or create a Firefox account
+- [ ] It should allow users to select and send one file at a time
+- [ ] It should limit the sendable file size to 1GB
+- [ ] It should allow users to set an expiration time of 5 minutes, 1 hour, or 24 hours
+- [ ] It should allow users to set an download count of 1, 10 or 20 downloads
+
+### Authenticated Users
+- [ ] It should indicate that the user is signed in via Firefox Account
+- [ ] It should allow the user to sign out
+- [ ] It should allow users to select and send multiple files in one URL
+- [ ] It should limit users to sending 2.5 GB per Send.
+- [ ] It should allow users to extend Send times up to 1 Week
+- [ ] It should allow users to extend Send download counts up to 200 times
+
+### Cards
+- [ ] It should display the name of the sent file/files
+- [ ] It should display the time remaining before expiration
+- [ ] It should display the number of downloads remaining before expiration
+- [ ] It should have a button that lets the user copy the send link to their clipboard
+- [ ] It should show a unique preview thumbnail that has some relationship to the file types or content being sent* (see 5.1 in spec)
```

Comment:
boo on thumbnail 👎

Reply:
Hey @dannycoates can you elaborate more about why thumbnail is a bad idea? thanks!

Reply:
I think it's basically impossible to extract an actual thumbnail, maybe we could do file type icons or stacks of icons...seems like it would be nice to have some kind of graphic element here?

Reply:
I see. Let's just use some generic icons here.